packages/stateManager/mod.ts

```diff
@@ -207,26 +212,32 @@ export default class StateManager {
     };
   }
 
-  #sendPatch(state: IStoredState, patch: Patch) {
-    // send patch to peers
+  async #sendPatch(state: IStoredState, patch: Patch) {
+    // send patch to connected peers
+    const allWrites: Promise<void>[] = [];
```

Comment:
this seemed more stable in the face of having no connections but idk really. Wdyt @wanderer ?